linkml/generators/fhirgen/mapping__enum_valueset.csv

```diff
@@ -0,0 +1 @@
+enum	valueset
```

Comment:
I hope I don't create any ire w/ this pattern. This has been a highly successful model mapping pattern to me. But it seems a bit redundant/recursive. Doesn't LinkML itself do model mappings in yml? Yet, I vastly prefer doing this in a tabular file (TSV / CSV). Could I do such a mapping in SchemaView? Ought those best be done in GoogleSheets and either pulled down from the cloud, and/or saved/cached as CSV/TSV?

Reply:
You'll have to enlighten me with how this works. Unless @cmungall has an opinion about it should be before we decide on this.